app/src/main/java/edu/akelael/comics/presenter/ComicListPresenter.java

```diff
@@ -0,0 +1,35 @@
+package edu.akelael.comics.presenter;
+
+import java.util.List;
+
+import edu.akelael.comics.model.ComicListModel;
+import edu.akelael.comics.model.IComicListModel;
+import edu.akelael.comics.model.IComicListModelCallbacks;
+import edu.akelael.comics.model.Marvel;
+import edu.akelael.comics.view.IComicListActivity;
+
+public class ComicListPresenter implements IComicListPresenter, IComicListModelCallbacks {
```

Comment:
IMO this class has more responsibilities than it should have. If this is a presenter, then there's no need for it to act also as a callback. I suggest using an anonymous object for the callback instead of making the presenter implement the callback interface.